flybywire-aircraft-a320-neo/html_ui/Pages/VCockpit/Instruments/Airliners/FlyByWire_A320_Neo/CDU/A320_Neo_CDU_WindPage.js

```diff
@@ -248,4 +287,101 @@ class CDUWindPage {
+                                });
+                                lastAltitude = altitude;
+                                altIdx++;
+                            }
+                        }
+                        // Now we add the closest wind data to the altitude of the clbWpt
+                        clbWpt.wind_data.level.forEach((wind, levelIdx) => {
+                            const altitude = parseInt(wind.altitude);
+
+                            let deltaPrevLevel = 0;
+                            let deltaThisLevel = 0;
+                            // Look backwards for the closest level
+                            if (levelIdx > 0 && levelIdx < clbWpt.wind_data.level.length - 1) {
+                                deltaPrevLevel = Math.abs(clbWpt.altitude_feet - parseInt(clbWpt.wind_data.level[levelIdx - 1].altitude));
+                                deltaThisLevel = Math.abs(clbWpt.altitude_feet - altitude);
+                            }
+
+                            // Check that altitude isn't backtracking
+                            if (altitude > lastAltitude) {
+                                const idx = (deltaPrevLevel > deltaThisLevel) ? levelIdx : levelIdx - 1;
+
+                                const idxAltitude = parseInt(clbWpt.wind_data.level[idx].altitude);
+                                const direction = parseInt(clbWpt.wind_data.level[idx].wind_dir);
+                                const speed = parseInt(clbWpt.wind_data.level[idx].wind_spd);
+
+                                // Check again that we didn't backtrack
+                                if (idxAltitude > lastAltitude) {
+                                    windData.push({
+                                        direction,
+                                        speed,
+                                        altitude: idxAltitude / 100,
+                                    });
+                                    lastAltitude = idxAltitude;
+                                }
+                            }
+                        });
+                    });
+                    mcdu.winds.climb = windData;
+                    break;
+                case "CRZ":
+                    const toc = mcdu.simbrief.navlog.find((val) => val.ident === "TOC");
+                    mcdu.winds.cruise = [];
+                    toc.wind_data.level.forEach((val) => {
+                        const direction = parseInt(val.wind_dir);
+                        const speed = parseInt(val.wind_spd);
+                        const altitude = parseInt(val.altitude / 100);
+                        mcdu.winds.cruise.push({
+                            direction,
+                            speed,
+                            altitude,
+                        });
+                    });
+                    break;
+                case "DSC":
+                    const tod = mcdu.simbrief.navlog.find((val) => val.ident === "TOD");
```

Comment:
I am under the impression that SimBrief provides wind data for the descent itself at different levels and not just at the T/D. Could we use those winds instead?